app/code/community/Reverb/ReverbSync/Helper/Data.php

```diff
@@ -281,6 +281,7 @@ protected function _getCurlResource($url, $options_array = array())
         $options_array[CURLOPT_SSL_VERIFYPEER] = 0;
         $options_array[CURLOPT_HEADER] = 0;
         $options_array[CURLOPT_RETURNTRANSFER] = 1;
+        $options_array[CURLOPT_USERAGENT] = 'Reverb-Magento MagentoVersion=community-1.9';
```

Comment:
Sounds good. As a note, if there are instances where customers are on one domain, but the admin panel is on another domain, then requests coming as a result of order placements will show a different domain than ones caused by updates done via the admin panel, or by calls made via cron execution